crates/bevy_render/src/render_phase/draw_state.rs

```diff
@@ -21,13 +21,14 @@ use wgpu::{IndexFormat, QuerySet, RenderPass};
 struct DrawState {
     pipeline: Option<RenderPipelineId>,
     bind_groups: Vec<(Option<BindGroupId>, Vec<u32>)>,
-    vertex_buffers: Vec<Option<(BufferId, u64)>>,
+    // BufferId, offset, size
+    vertex_buffers: Vec<Option<(BufferId, u64, u64)>>,
```

Comment:
```suggestion
    /// List of vertex buffers by [`BufferId`], offset, and size. See [`DrawState::buffer_slice_key`]
    vertex_buffers: Vec<Option<(BufferId, u64, u64)>>,
```

Might as well make this a documentation so that it appears as a tooltip in most IDEs.